Makefile

```diff
@@ -1,17 +1,25 @@
-.PHONY: build-linux
+local:
+	cargo build --release
+
+run:
+ifndef ARGS
+	@echo Run "make run" with ARGS set to pass arguments…
+endif
+	cargo run --release -- $(ARGS)
+
 build-linux:
 	cargo build --target x86_64-unknown-linux-musl --release
 	strip target/x86_64-unknown-linux-musl/release/miniserve
 	upx target/x86_64-unknown-linux-musl/release/miniserve
 
-.PHONY: build-win
 build-win:
 	RUSTFLAGS="-C linker=x86_64-w64-mingw32-gcc" cargo build --target x86_64-pc-windows-gnu --release
 	strip target/x86_64-pc-windows-gnu/release/miniserve.exe
 	upx target/x86_64-pc-windows-gnu/release/miniserve.exe
 
-.PHONY: build-apple
 build-apple:
 	cargo build --target x86_64-apple-darwin --release
 	strip target/x86_64-apple-darwin/release/miniserve
 	upx target/x86_64-apple-darwin/release/miniserve
+
+.PHONY: local run build-apple build-win build-linux
```

Comment:
I try to conform to Clark Grubb's Makefile style guide. See [here](http://clarkgrubb.com/makefile-style-guide#phony-targets).

Reply:
Ah, OK. I'll revert my commit but I'll have to wait until I'm done with some other work to do tonight. I wish GitHub's interface had a revert 😩